src/Taweret/utils/utils.py

```diff
@@ -243,13 +247,15 @@ def mixture_function(
         w = np.array(w).flatten()
         return w, 1 - w
     elif method == 'beta':
-        print('Warning: mixture_function - the `beta` choice forces a stochastic')
-        print('         likelihood to be returned after calibration')
+        print('Warning: mixture_function - the `beta` choice forces a \
```

Comment:
@asemposki The long text was originally split between two `print` lines.  Since flake8 already forced you to do a line continuation in the first `print`, can you just combine the two `print` statements?

See below as well.

Reply:
I can do that.

Reply:
Thanks.  I'll let you know when I finish my review so that you can do that work during your review.